services/brig/src/Brig/Provider/API.hs

```diff
@@ -265,20 +273,31 @@ activateAccountKey key val = do
       lift $ sendApprovalConfirmMail name email
       pure . Just $ Public.ProviderActivationResponse email
 
-getActivationCodeH :: (Member GalleyAPIAccess r, Member VerificationCodeSubsystem r) => EmailAddress -> (Handler r) Code.KeyValuePair
+getActivationCodeH ::
```

Comment:
We could drop this `wai-routes` convention of ending handler names with "H" now that you've already changed the line.